### DISCUSSION TITLE
Quick info

### DISCUSSION THREAD

**Discussion by sou-joshi** [heybereket/oasis#6] [General]
Hello,
Would like to understand how is it different from up for grabs/Issue hub etc? What makes it a goto in comparison to these?

**Comment by heybereket** [comment]
Hey there! 👋🏻

Oasis has a goal of making good open source repositories to contribute to, easier and more accessible. 
It's like basically like connecting people who want to contribute to, with people who want contributors. GitHub has millions of repositories and if you want help handling issues, it can get difficult as you wouldn't know where to go, same as if you wanted to contribute to repositories. 

Let me know if you have any further questions!

**Reply by sou-joshi** [reply]
Cool, that's exactly what the below mentioned sites do. https://up-for-grabs.net/#/
http://issuehub.pro/?label%5B%5D=help+wanted&language=

Wanted to understand how is your project different to these.

**Reply by marques-kevin** [reply]
https://opencollective.com/discover can help to discover open source projects too. 

I think there will never be too much websites that helps people to find projects to contribute. As an open-source contributor, the sites you mentionned was unknown to me. And Oasis can be more than a Google for open source projects. Oasis could be a website that showcase the Open Source Contributors, or something else :)

**Reply by heybereket** [reply]
We're planning to scale massive, our goal isn't to become the next Google/GitHub but instead to help people find good projects to contribute to.
